torch/csrc/onnx/init.cpp

```diff
@@ -28,6 +28,7 @@ void initONNXBindings(PyObject* module) {
     .value("ONNX", OperatorExportTypes::ONNX)
     .value("ONNX_ATEN", OperatorExportTypes::ONNX_ATEN)
     .value("ONNX_ATEN_FALLBACK", OperatorExportTypes::ONNX_ATEN_FALLBACK)
-    .value("RAW", OperatorExportTypes::RAW);
+    .value("RAW", OperatorExportTypes::RAW)
+    .value("ONNX_PBTXT", OperatorExportTypes::ONNX_PBTXT);
```

Comment:
it won't run Aten fallback mode through it, but I guess it's fine. @jamesr66a to confirm in case we want to run the entire LATTE in this mode. If yes - we can make it a bitflag

Reply:
Can we use `export_to_pretty_string` https://github.com/pytorch/pytorch/blob/5e5c15dd426ae608868ea76ec5be2d8817a31523/torch/onnx/utils.py#L210 instead of making a new mode?